app/src/renderer/components/monitor/PageValidatorIndex.vue

```diff
@@ -1,26 +1,9 @@
 <template lang="pug">
-page(icon="storage" :title="validator.pub_key.data")
+page(title="Validator Profile")
   div(slot="menu"): tool-bar
     router-link(to="/validators" exact)
       i.material-icons arrow_back
       .label Back
-
```

Comment:
so this will be a completely empty page? is this one of the components we don't use anymore?

Reply:
i will delete this file.